src/__tests__/createTask.js

```diff
@@ -0,0 +1,50 @@
+import { jsxEmptyExpression } from '@babel/types'; // eslint-disable-line
```

Comment:
- [ ] This file should be called `[File_name].test.js` in order to work properly, please change this. 😄